lib/matplotlib/backend_bases.py

```diff
@@ -1189,19 +1219,20 @@ class TimerBase(object):
           object should call, which will handle the task of running
           all callbacks that have been set.
 
-    Attributes:
+    Attributes
+    ----------
+    interval : scalar
+        The time between timer events in milliseconds. Default is 1000 ms.
 
-        * `interval`: The time between timer events in
-          milliseconds. Default is 1000 ms.
+    single_shot: bool
```

Comment:
Here and below, my understanding from @nellev is that the colon must be surrounded by spaces.